src/components/Contentful/SectionWithBackgroundClassic.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+	<section class="">
+		<div class="tw-relative tw-p-0">
+			<div
+				class="tw-relative tw-w-full tw-overflow-hidden tw-z-1 tw-m-0 tw-top-0"
```

Comment:
minor, I don't think the `tw-m-0` is affecting anything

Reply:
removed